builder/Cargo.toml

```diff
@@ -5,10 +5,20 @@ rust-version.workspace = true
 edition.workspace = true
 version.workspace = true
 license.workspace = true
-publish = false
 
 [features]
-default = []
+# Default features to release
+default = [
+  "crashtracker",
+  "profiling",
+  "telemetry",
+  "data-pipeline",
+  "symbolizer",
+  "library-config",
+  "log",
+  "ddsketch",
+  "ffe",
+]
```

Comment:
The builder was also thougt for external SDKs which  can pull it and build a tailored library based on their needs. Therefore re-exporting all by default defies that purpose a bit. That was the reason we were setting all features in each script. That said, since you are in a hurry we can leave it that way and tackle how set the features globally for our release model in another PR.